api/package.json

```diff
@@ -44,7 +44,7 @@
     "jsonwebtoken": "^9.0.0",
     "jwks-rsa": "3.1.0",
     "lodash": "^4.17.21",
-    "minio": "^7.1.0",
+    "minio": "^7.1.1",
```

Comment:
We can only use the minio version v7.11, the latest v7.1.3 will return the following error in local. It won't block the tests and just because of some console log, feel it's okay to keep a lower version for now. 
<img width="772" alt="Screen Shot 2024-01-03 at 4 48 42 PM" src="https://github.com/bcgov/nr-fom/assets/23131735/b79eddc2-3abd-4999-976a-d33cbd7e0982">